content/source/docs/enterprise/private/logging.html.md

```diff
@@ -8,7 +8,32 @@ sidebar_current: "docs-enterprise2-private-logging"
 
 This document contains information about interacting with Private Terraform Enterprise logs. There are two types of logs, application logs and audit logs. Application logs emit information about the services that comprise Terraform Enterprise. Audit logs emit information whenever any resource managed by Terraform Enterprise is changed.
 
-## Application Logs - AMI based installs
+## Application Logs 
+
+### Installer
+
+The installer-based version runs the application in a set of docker containers. As such, any tooling that can interact with docker logs
+can read the logs. This includes running the command `docker logs`, as well as access the [Docker API](https://docs.docker.com/engine/api/v1.36/#operation/ContainerLogs).
```

Comment:
I realize this is just a copy of the original content, but I think we can improve the wording:

> Installer-based PTFE runs in a set of Docker containers. As such, any tooling that can interact with Docker logs can read the logs. This includes the command `docker logs`, as well as access via the [Docker API].